webapp/components/PollutionMapBase.tsx

```
@@ -52,6 +69,47 @@ export default function PollutionMapBaseLayer({
+          newMapCenter = [-61, 14.7];
+          newZoomFactor = 9;
+          break;
+        case ZONE_MAYOTTE:
+          newMapCenter = [45, -12.75];
+          newZoomFactor = 9;
+          break;
+        case ZONE_GUYANE:
+          newMapCenter = [-52.7, 4.3];
+          newZoomFactor = 6;
+          break;
+        case ZONE_METROPOLE:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)